### PR TITLE
Threw a dryer sheet into the code.

### PR DESCRIPTION
**Pull request by freakboy3742**
(i.e., removed some code lint)

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-multi_line_output = 5
-not_skip = __init__.py
-known_standard_library = marshal
-known_first_party = voc
-sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+[flake8]
+# https://flake8.readthedocs.org/en/latest/
+exclude=settings.py,wsgi.py,manage.py,*/migrations/*
+max-complexity = 10
+max-line-length = 119
```
Comment (isagalaev) [inline setup.cfg]: Just wonder, why not a round number? :-)

Comment (freakboy3742) [inline setup.cfg]: Habit, mostly. IIRC, editors in the Good Ole' Days used to show a continuation mark in the 80th column, so you set the line length at 79, and if you ever saw that mark, you knew you had a problem.

File: setup.cfg [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+[metadata]
+description-file = README.md
+[pep8]
```
Comment (Mariatta) [inline setup.cfg]: `pep8` was renamed into `pycodestyle`
So.. maybe use `pycodestyle` instead? :smiley:
http://pycodestyle.readthedocs.io/en/latest/
I recall reading a discussion where it was proposed that `pep8` be removed from pypi, but I don't know what's the outcome of that.

File: setup.cfg (hunk 1 above)
Comment (isagalaev) [inline setup.cfg]: Make sense :-) Thanks for the explanation!

**Pull request merged**